src/cli/cli_coap_secure.cpp

```diff
@@ -586,30 +586,26 @@ void CoapSecure::HandleClientResponse(otMessage *aMessage, const otMessageInfo *
 }
 
 #if CLI_COAP_SECURE_USE_COAP_DEFAULT_HANDLER
-void OTCALL CoapSecure::DefaultHandle(void *               aContext,
-                                      otCoapMessage *      aHeader,
-                                      otMessage *          aMessage,
-                                      const otMessageInfo *aMessageInfo)
+void OTCALL CoapSecure::DefaultHandle(void *aContext, otMessage *aMessage, const otMessageInfo *aMessageInfo)
```

Comment:
nit: let's rename this to `CoapSecure::DefaultHandler()` to be consistent with the rest of OpenThread.